glean-core/android/src/test/java/mozilla/telemetry/glean/private/CounterMetricTypeTest.kt

```diff
@@ -56,25 +56,6 @@ class CounterMetricTypeTest {
         assertEquals(11, counterMetric.testGetValue())
     }
 
-    @Test
-    fun `counters with no lifetime must not record data`() {
-        // Define a 'counterMetric' counter metric, which will be stored in "store1".
-        // It's disabled so it should not record anything.
-        val counterMetric = CounterMetricType(
-            disabled = true,
-            category = "telemetry",
-            lifetime = Lifetime.Ping,
-            name = "counter_metric",
-            sendInPings = listOf("store1")
-        )
-
-        // Attempt to increment the counter
-        counterMetric.add(1)
-        // Check that nothing was recorded.
-        assertFalse("Counters must not be recorded if they are disabled",
-            counterMetric.testHasValue())
-    }
```

Comment:
This test is *exactly* the same as the one below it (except the name and the comment), so I just went ahead and removed it.  Save a few cycles per test ;)